frontend/public/components/utils/datetime.ts

```diff
@@ -1,9 +1,15 @@
 import * as _ from 'lodash-es';
 import * as moment from 'moment';
 
+const getLocale = () => localStorage.getItem('bridge/language') || 'en';
+
+export const getLocaleDate = (date: Date, options): string =>
+  date.toLocaleDateString(getLocale(), options);
+
```

Comment:
The `getLocale` approach isn't going to cover people who haven't used the language selector. We only set `bridge/language` when the selector is used. We also auto-detect languages, so you could wind up with people seeing console in Japanese, but with these timestamps in English. 

When we had to do this elsewhere, we filtered the `i18next.languages` array against our list of supported languages. You can see this in action in `language-preferences-modal.tsx.`

We're also using Moment to format general timestamps in case that seems like it won't work for what you need here.

Reply:
I don't see a reason not show dates in the user's locale even if we haven't translated the console to that language. We should use the browser's locale without filtering for supported languages.

Reply:
I've modified this PR to default to `undefined` for the locale instead of `en`. Then `toLocaleDateString` will default to the browser's locale if no language preference has been set. WDYT?

I guess we should aim to avoid using Moment here since it is now semi-deprecated: https://momentjs.com/docs/#/-project-status/

Reply:
@kyoto Yeah, that's what I was thinking. I opened https://issues.redhat.com/browse/CONSOLE-2790 to look at removing moment.js.